<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordEntryDialog</class>
 <widget class="QDialog" name="PasswordEntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Title</string>
  </property>
  <widget class="QPushButton" name="edit_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Edytuj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>391</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>153</y>
     <width>399</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa użytkownika lub email</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_line_edit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>253</y>
     <width>399</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>226</y>
     <width>399</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hasło</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>93</y>
     <width>399</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_line_edit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>113</y>
     <width>399</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_line_edit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>399</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
